src/printer.js

```diff
@@ -3094,6 +3094,15 @@ function printAssignment(printedLeft, operator, rightNode, printedRight, options
       options.tabWidth,
       concat([line, printedRight])
     );
+  } else if (
+    rightNode.type === "StringLiteral" ||
+    (rightNode.type === "Literal" &&
+      typeof rightNode.value === "string")
+  ) {
+    printed = indent(
+      options.tabWidth,
+      concat([line, printedRight])
```

Comment:
Couldn't this be collapsed with the previous `else if`? The contents are the same code.